ci/desktop.groovy

```diff
@@ -187,6 +187,8 @@ def bundleMacOS(type = 'nightly') {
     sh 'chmod +x Status.app/Contents/Resources/ubuntu-server'
     sh 'cp ../desktop/bin/StatusIm Status.app/Contents/MacOS/Status'
     sh 'cp ../desktop/reportApp/reportApp Status.app/Contents/MacOS'
+    sh "cp ../deployment/.env  Status.app/Contents/Resources"
+    sh 'ln -sf ../Resources/.env Status.app/Contents/MacOS/.env'
```

Comment:
Personally I'd prefer that there are no hidden files under `deployment` and Resources, so I'd go with:
```
sh 'cp ../deployment/env  Status.app/Contents/Resources'
sh 'ln -sf ../Resources/env Status.app/Contents/MacOS/.env'
```
...and of course everywhere else `.env` is referenced.

Reply:
@gnl , makes sense, thanks!

Reply:
But why not just name the file "env" and set the appropriate file path to be loaded on app start.